guides/v2.1/cloud/project/new-relic.md

```diff
@@ -18,13 +17,46 @@ New Relic APM provides the following features to Magento:
 -  **App Map**—View all application dependencies within your Magento site, extensions, and external services.
 -  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event.
 
-Your {{site.data.var.ece}} account includes the agent software for New Relic; you do not need to purchase or install the New Relic extension (different than the APM service). You receive your credentials and license information from Magento during launch and can access your license key in the _Project Web Interface_ by clicking **View Details** for your project. The **NewRelic Service** section includes your Account Number, License Key, and other access keys.
+By default, the Pro Staging and Production environments include New Relic. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation. {{site.data.var.ece}} Pro plan customers also need to submit a support ticket to use New Relic on an Integration branch. 
+
+
+{:.bs-callout .bs-callout-info}
+Your Magento Commerce Cloud account includes the software for the New Relic APM agent. You do not need to purchase or install the New Relic extension, which is different than the New Relic APM service. 
+
+## New Relic account credentials
+
+When Magento provisions your {{site.data.var.ece}} project, the Project Owner should receive an email from New Relic with the credentials and instructions for accessing the account. If you did not receive the email, you can use the Project Owner email address to [reset the password](https://rpm.newrelic.com/forgot_password){:target="_blank"}.
+
+If you cannot access the account, submit a [Magento support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html).
+
+
+## Add account users
+
+{{site.data.var.ece}} Project Owners and Admin users can add and remove users from the New Relic account.
+
+We recommend adding at least one account through New Relic to manage all access, integrations, and usage of the tool. We also recommend promoting one of the added members to Admin, to manage all New Relic access and integrations.
```

Comment:
one "user" account...

There are many references to "account" , so I think it would help to be more specific. Earlier, the subject is about the overall New Relic account, and then there is discussion about creating at least one account. I'm sure it is to create at least one user account, but it helps to be specific.